test/python/test_cuda_vmm_utils.py

```
@@ -0,0 +1,103 @@
+            deadline = time.monotonic() + 10.0
+            while state == "PROC" and time.monotonic() < deadline:
+                state = agent1.check_xfer_state(handle)
+                assert state in ("DONE", "PROC")
+            assert state == "DONE"
+
+            notif_done = False
+            while time.monotonic() < deadline:
+                notif_done = agent2.check_remote_xfer_done(agent1.name, b"fabric_done")
+                if notif_done:
+                    break
+                pass
```

Ruff (0.15.17)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)